modules/tester/tester.go

```diff
@@ -0,0 +1,18 @@
+package tester
+
+import (
+	"os"
+	"path/filepath"
+)
+
+const (
+	SiaTestingDir = "SiaTesting"
+)
+
+// TempDir returns the tempified version of the input directory, after creating
+// the tempified directory.
+func TempDir(dir string) string {
+	temp := os.TempDir()
+	tempified := filepath.Join(temp, SiaTestingDir, dir)
+	return tempified
```

Comment:
This should probably be one line lol.

My guess is that os.TempDir also is just a global variable. I don't think there's a huge reason to not make the function call every time.